src/mutations/authentication.test.ts

```diff
@@ -557,7 +538,7 @@ describe('Authentication Mutations', () => {
   //     (utils.getStoredSessions as jest.Mock).mockReturnValue(MOCK_SESSIONS);
```

Comment:
Is this commented code useful or should we remove it ?

Reply:
I think we keep it because this was the user switching feature that we had before. We would like to add it back but we need to think about how to do it properly with the httpOnly cookies.